src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -751,7 +751,8 @@ private void DynamoViewModelChangeScaleFactor(object sender, EventArgs e)
                     dynamoViewModel.ScaleFactorLog = view.SliderValue;
                     dynamoViewModel.CurrentSpace.HasUnsavedChanges = true;
 
-                    dynamoViewModel.ExecuteCommand(new DynamoModel.ForceRunCancelCommand(false, false));
+                    var allNodes = dynamoViewModel.HomeSpace.Nodes;
+                    dynamoViewModel.HomeSpace.MarkNodesAsModifiedAndRequestRun(allNodes, forceExecute: true);
```

Comment:
I think `MarkNodesAsModifiedAndRequestRun` takes in nodes as input so that one can choose which nodes to pass to mark as modified.